dist/common/scripts/scylla_util.py

```diff
@@ -99,6 +99,19 @@ def scyllabindir():
 def sysconfdir():
     return str(sysconfdir_p())
 
+def get_required_aio_slots():
+    return str(cpu_count() * 11026 + 65536)
+
+def configure_aio_slots():
+    aio_max_nr_value = "fs.aio-max-nr = " + get_required_aio_slots()
+    with open("/etc/sysctl.conf", "r+") as file:
```

Comment:
We shouldn't be touching sysctl.conf, that's what sysctl.d is for. And we already configure it. We should drop a file into dist/common/sysctl.d.

Reply:
@avikivity Already changed so we will use `/etc/sysctl.d/aio-max-nr.conf` file instead 

Will push the changes soon

Reply:
ping